tests/Propel/Tests/Generator/Command/InitCommandTest.php

```diff
@@ -48,6 +48,10 @@ public function setUp()
 
     public function testExecute()
     {
+        if (!method_exists(CommandTester::class, 'setInputs')) {
+            $this->markTestSkipped('Interactive console input was not present in some earlier versions of symphony/console');
```

Comment:
It should be `symfony/console`. If the code is only working with newer versions of the console shouldn't we drop versions before that feature was introduced?

Reply:
```suggestion
            $this->markTestSkipped('Interactive console input was not present in some earlier versions of symfony/console');
```

> It should be symfony/console. If the code is only working with newer versions of the console shouldn't we drop versions before that feature was introduced?

Yes, but I think that makes sense in major release because we have a lot of package consumers using the old version.